assets/js/src/core/modules/element/dynamic-types/definitions/objects/data-related/components/many-to-many-relation/grid.tsx

```diff
@@ -94,7 +95,20 @@ export const ManyToManyRelationGrid = forwardRef(function ManyToManyRelationGrid
             editable: false,
             config: getElementCellConfig(props.inherited === true || props.disabled === true)
           },
-          size: 200
+          size: 200,
+          cell: (info) => {
+            return (
+              <Flex
+                align={ 'center' }
+                className={ 'p-mini' }
+              >
+                <Tag
+                  bordered={ false }
+                  color="geekblue"
+                ><SanitizeHtml html={ info.getValue() ?? '' } /></Tag>
```

Comment:
This works for many-to-many-relations but unfortunately it seems that we need to find other locations where we need to apply this for the other multi-relation data types:

![image](https://github.com/user-attachments/assets/378f07e4-be97-42d1-a18c-c170d603101b)